mtp_noms_ops/templates/security/credits_list.html

```diff
@@ -94,9 +94,19 @@
     {% endif %}
 
   {% if is_search_results %}
+    <p>
     {% with search_form_item=breadcrumbs|slice:":-1"|last %}
       <a href="{{ search_form_item.url }}" class="button">{% trans 'Search again' %}</a>
     {% endwith%}
+    </p>
+
+    {% if all_prisons_simple_search_link %}
+    <p>
+      <a href="{{ all_prisons_simple_search_link }}">
+        {{ form.search_description.description }} {% trans 'from all prisons' %}
```

Comment:
when we come to look at jira story MTP-1081, i think it might be worth moving the "from all prisons" bit into the mechanism that generates descriptions in form classes

Reply:
I agree, I changed that line and added the extra text to the generated description but it might need more refactoring when starting story 1081.